### PR TITLE
Update ruby version. Add pronto eslint runner.

### PR DESCRIPTION
**Pull request by Saicheg**
* Bump ruby version to 2.6 version 
* Add missing `erb_lint` runner
* Add new `eslint_npm` linter

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,27 +1,33 @@
-FROM ruby:2.5
+FROM ruby:2.6-buster
 
 LABEL maintainer="Josh Bielick <jbielick@adwerx.com>"
 
+ENV BUNDLER_VERSION="2.1.4"
+
+RUN apt-get update && apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
```
Comment (jbielick) [inline Dockerfile]: I had originally intended for this action to only use ruby runners. Would running eslint be better served by a different github action?

File: Dockerfile [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
   cmake \
   git \
   openssl \
   yamllint \
+  nodejs \
   && rm -rf /var/lib/apt/lists/*
 
-RUN gem install bundler
+RUN gem install bundler --version "${BUNDLER_VERSION}"
+RUN npm install -g eslint
```
Comment (jbielick) [inline Dockerfile]: We would probably want to pin the eslint version that's installed.

File: Dockerfile (hunk 1 above)
Comment (Saicheg) [reply Dockerfile]: I think pronto is very well-know in Ruby community as a way to run different linters. And big part of Ruby community is using Rails for their everyday development. 

Rails is not only focusing on giving a great experience for backend development, but also includes nice tools to deal with frontend. And javascript is a kind of a king in the frontend world. That's why I think it is important to give developers the ability to fully lint their Rails projects. 

Also, if we are only focusing on ruby linting here, I would say that `pronto-scss` should not be a part of current setup as well.

Talking about sizes and numbers:

```
Image with nodejs + eslint included
<none>                                                           <none>              c86b7ed25447        About a minute ago   1.23GB
```
```
Image without nodejs + eslint included
docker images | grep eb84dcedcd92
<none>                                                           <none>              eb84dcedcd92        6 seconds ago       1.1GB
```

So `nodejs` + `eslint` only increases image size by ~120MB, which I don't think is a dramatic increase.

I want to highlight that currently, we are using `ruby:2.6-buster` with size of 840MB. If we want to optimize image size - we definitely should consider using `ruby:2.6-slim` with 147MB. If you want - I am happy to help you with that.

Comment (jbielick) [reply Dockerfile]: `2.6-slim` sounds like a good compromise. 

What I worry about is how many ways there are of running / configuring `eslint` and issues showing up on this repo for things I can't do anything about. For instance, how does `pronto-eslint_npm` handle eslint config `extends`? For those you need `node_modules` installed so eslint can load the extra configuration from there. Right off the bat this wouldn't work for the Rails applications we run—hence my hesitancy.

I think as long as we're not installing python or php this should be okay.

If you update to 2.6-slim I think you'll need `pkg-config` installed.

Comment (Saicheg) [reply Dockerfile]: @jbielick interesting, but I did try with our company config and `ESLINT_CONFIG` and it did work for me. I guess `extend` should work OK if they are on the same folder.

Please check updated version of Dockerfile, now it is 844MB~

**Comment by Saicheg** [comment]
@jbielick thanks for merging that! 
FYI - it looks like latest build failed on master

**Comment by jbielick** [comment]
@Saicheg 👍 
yeah, docker hub is _really_ delayed so I cancel to build manually.

**Comment by jbielick** [comment]
https://github.com/AdWerx/pronto-ruby/releases/tag/v2.5

**Pull request merged**